packages/vue-components/src/panels/NestedPanel.vue

```diff
@@ -226,8 +236,10 @@ export default {
     .card-title {
         display: inline-block;
         font-size: 1em;
+        line-height: 1.2em;
```

Comment:
Just an opinion: 

I think the card titles look slightly more centered vertically without setting the `line-height` here.

With `line-height`:
![image](https://user-images.githubusercontent.com/47470981/150747218-b4c55054-beab-4b07-a7df-0468f63a3481.png)


Without `line-height`:
![image](https://user-images.githubusercontent.com/47470981/150747151-b95e487d-244c-4830-b5fb-7e5e2c80bc6e.png)

Reply:
I see what you mean, but it helps for panels with long names.

With line-height:
<img width="448" alt="With line-height picture" src="https://user-images.githubusercontent.com/59471320/152721230-ac149beb-c370-49fc-b1bf-25f98beee130.png">

Without line-height:
<img width="439" alt="Without line-height picture" src="https://user-images.githubusercontent.com/59471320/152721296-7c2baf7d-da8b-4c72-adf7-fd4949a59c4c.png">

Reply:
Ahh alright then can keep it as it is for now :)